zarp/run/snakemake.py

```diff
@@ -0,0 +1,49 @@
+"""Module for running snakemake."""
+
+import subprocess
+import os
+
+from zarp.config.models import Run
+
+
+class SnakemakeExecutor:
+    """Run snakemake with system calls."""
+
+    def __init__(self, run: Run) -> None:
+        self.run_dict = run.dict()
+        self.success = None
+
+    def prepare_run(self, snkfile: str, workdir: str) -> None:
+        """Configure list of strings for execution.
+        """
+        run_list = ["snakemake"]
+        # Snakemake config
+        run_list.extend(["--snakefile", snkfile])
+        run_list.extend(["--cores", str(self.run_dict["cores"])])
+        # run_list.extend(["--config", f"workdir={workdir}"])
+        # execution profile
+        if self.run_dict['execution_profile'] == "local-conda":
+            prof = ["--profile", os.path.join("submodules", "zarp", "profiles", "local-conda")]
```

Comment:
I'd recommend to use the excellent `pathlib` module for any path operations in Python3. It's really much better than the older `os.path` module.

Reply:
I'll have a look into this, but not my priority right now :-)